sg-auto-release.js

```diff
@@ -54,6 +54,49 @@ function surgeDeploy({sourceDirectory, deployDomain, rootPath}) {
 
 }
 
+async function addCommentIfNotExist({repo, pr, githubToken, message}) {
+  if (await gitGetAllComments({repo, pr, githubToken, message})) {
+    gitAddComment({repo, pr, githubToken, message});
+  } else {
+    console.log('skipping adding comment - already exist');
+  }
+}
+
+function gitGetAllComments({repo, pr, githubToken, message}) {
```

Comment:
the function name is not demonstrating what it is actually doing.
It resolves boolean value if there are 0 comments, so I would rename to hasComments or something